experiences.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Experiences</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header><h1>Experiences</h1></header>
+
+    <nav>
+        <a href="home.html">Home</a>
+        <a href="bio.html">Bio</a>
+        <a href="portfolio.html">Portfolio</a> 
```

Comment:
while I see the benefit of having the nav bar change to different nav items depending on what page you're on, I feel like an end user might not pick up on this immediately - (I didn't, and almost missed your fun surprise on one of the pages heh!) Consider a navbar with consistent items so that any page can be easily reached at any point throughout the site.